### DISCUSSION TITLE
默认 Tab 选择

### DISCUSSION THREAD

**Discussion by FoundTheWOUT** [the1812/Bilibili-Evolved#3100] [提问 / Q&A]
我可以选择默认 Tab 吗。或者记录我上一次停留的 Tab ？
![image](https://user-images.githubusercontent.com/47922588/157577159-c2987a87-9cea-4469-b283-4d0c35d7564b.png)

**Comment by the1812** [comment]
可以参考 https://github.com/the1812/Bilibili-Evolved/blob/preview-fixes/registry/lib/plugins/settings-panel/recent-components/index.ts 写个插件
